C-Plus-Plus/math/Euler's_Totient_function.cpp

```diff
@@ -0,0 +1,38 @@
+// Euler's Totient Function represented by F(n)
+// denotes the no. of coprime of n from 1 to n
+#include <bits/stdc++.h>
+using namespace std;
+int F(int n){
+    int arr[n + 1], i, j;
+    arr[0]=0, arr[1]=1;     // as 1 is coprime with itself
```

Comment:
Add space between operator and operand in line 7, 8, 9, 11, 12, 13. Add cout<<"Enter a number : "; after line 22.

Reply:
@atarax665 Updated file, added cout<<"Enter a integer: ", added space between operator and operand as mentioned in above lines.

Reply:
Voila! LGTM :+1: